Tests/MongoSwiftSyncTests/SpecTestRunner/TestOperations/CollectionOperations.swift

```diff
@@ -460,20 +460,14 @@ struct ReplaceOne: TestOperation {
 }
 
 struct RenameCollection: TestOperation {
-    let session: String?
     let to: String
 
     func execute(
         on collection: MongoCollection<BSONDocument>,
-        sessions: [String: ClientSession]
+        sessions _: [String: ClientSession]
     ) throws -> TestOperationResult? {
-        let databaseName = collection.namespace.db
-        let cmd: BSONDocument = [
-            "renameCollection": .string(databaseName + "." + collection.name),
-            "to": .string(databaseName + "." + self.to)
-        ]
-        let reply = try collection._client.db("admin").runCommand(cmd, session: sessions[self.session ?? ""])
-        return TestOperationResult(from: reply)
+        _ = try collection.renamed(to: self.to)
```

Comment:
we should be passing in `sessions[self.session ?? ""]` for the session here as we were before when running this via `runCommand`